lomond/mask.py

```diff
@@ -10,77 +10,24 @@
 import six
 
 
-try:
-    from wsaccel.xormask import XorMaskerSimple
-except ImportError:
-    XorMaskerSimple = None
+make_masking_key = partial(os.urandom, 4)
 
 
-make_masking_key = partial(os.urandom, 4)
+if six.PY2:
+    _XOR_TABLE = [b''.join(chr(a ^ b) for a in range(256)) for b in range(256)]
+else:
+    _XOR_TABLE = [bytes(a ^ b for a in range(256)) for b in range(256)]
 
 
-if XorMaskerSimple is not None:
-    # Fast C version (works with Py2 and Py3)
-    def mask_payload(masking_key, data):
-        return XorMaskerSimple(masking_key).process(data)
+def mask_payload(masking_key, data):
+    """XOR mask bytes.
```

Comment:
Apologies to @toudi for deleting his lengthy explanation, but it was hard to maintain.